core/sql/regress/core/EXPECTED002.LINUX

```diff
@@ -464,22 +478,16 @@ A            AMAX         B                   BMAX                C            C
 >>
 >>
 >>------------------------------------------------------------------------
->>?section Genesis_10_000222_6892_p1
->>SELECT 1 FROM T002T3 T1
-+>GROUP BY T1.A
-+>HAVING T1.A >ANY
-+>  ( SELECT 2 FROM T002T1 T2
-+>    WHERE T2.C >SOME
-+>      ( SELECT AVG (T1.A) FROM T002T1 T3 )
-+>  );
-
-(EXPR)
-------
-
-     1
-     1
-
---- 2 row(s) selected.
+>>-- This test disabled since it is non -deterministic.
```

Comment:
Unless you plan to work on this right away it might be good to file a JIRA to keep track of this.

Reply:
Sure, I can do that.

Reply:
JIRA filed. See https://issues.apache.org/jira/browse/TRAFODION-2972